### PR TITLE
[14.0][MIG] mail_server_relay_disallowed

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
Standard migration to 14.0.

cc @AdriaGForgeFlow @MateuGForgeFlow

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Having the new system parameter for not doing spoofing, this module is not needed IMO.

**Comment by AaronHForgeFlow** [comment]
What's that new system parameter? Thanks in advance :innocent:

**Comment by ronozoro** [comment]
> What's that new system parameter? Thanks in advance 😇

If you want all your outgoing messages to use the same “From” address, use the key `mail.force.smtp.from` and set that address as value (such as `outgoing@mycompany.example.com`).

File: mail_server_relay_disallowed/models/ir_mail_server.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+            # exact name and address
+            (oldname, oldemail) = parseaddr(message["From"])
+            # use original name with new address
+            newfrom = formataddr((oldname, smtp.user))
+            # need to use replace_header instead '=' to prevent
+            # double field
+            message.replace_header("From", newfrom)
+            smtp.sendmail(smtp.user, smtp_to_list, message.as_string())
+            # END OF CODE ADDED
```
Comment (thomaspaulb) [inline mail_server_relay_disallowed/models/ir_mail_server.py:120]: I didn't know about the new Odoo configuration parameter, but for this purpose I always use [mail_outbound_static](https://github.com/OCA/social/tree/14.0/mail_outbound_static)... in comparison to that, why would this module be better?

**Pull request closed without merging**